app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontAddToCartFromCmsPageUsingAjaxActionGroup.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontAddToCartFromCmsPageUsingAjaxActionGroup">
```

Comment:
Hi @Usik2203. One of the final things with the tests I would ask you is to split this action group into 3 different action groups. 

- `StorefrontAddToCartFromCmsPageUsingAjaxActionGroup`
- Another action group for checking success message. We already have such [action group](https://github.com/magento/magento2/blob/18c8f31b6467e410bb94ab6b3fa2733dd024024e/app/code/Magento/Customer/Test/Mftf/ActionGroup/AssertStorefrontCustomerMessagesActionGroup.xml#L11).
- Another action group for checking minicart QTY. You may use the following action group 
https://github.com/magento/magento2/blob/d5cd1a487935dfd1c9adfd989b647721cc87dfdb/app/code/Magento/SalesRule/Test/Mftf/ActionGroup/AssertStorefrontMiniCartItemsActionGroup.xml#L10

Sorry for bothering you with such a big number of fixes. But once you get familiar with MFTF practices it will be much faster.

Thank you!

Reply:
Hi @rogyar . 
You don't bothering me by remarks.  
I have applied your recommendations. 
Thank you for code review !